### ISSUE TITLE
virtualbox package cannot download installer due to 404

### ISSUE THREAD

**Issue by rgl** [chocolatey-community/chocolatey-coreteampackages#1415]
## Current Behavior

```
You have virtualbox v6.1.0 installed. Version 6.1.2 is available based on your source(s).
virtualbox is not running
Progress: Downloading virtualbox 6.1.2... 100%

virtualbox v6.1.2 [Approved]
virtualbox package files upgrade completed. Performing other installation steps.
Attempt to get headers for https://download.virtualbox.org/virtualbox/6.1.2/VirtualBox-6.1.2-135662-Win.exe failed.
  The remote file either doesn't exist, is unauthorized, or is forbidden for url 'https://download.virtualbox.org/virtualbox/6.1.2/VirtualBox-6.1.2-135662-Win.exe'. Exception calling "GetResponse" with "0" argument(s): "The remote server returned an error: (404) Not Found."
Downloading virtualbox 64 bit
  from 'https://download.virtualbox.org/virtualbox/6.1.2/VirtualBox-6.1.2-135662-Win.exe'
ERROR: The remote file either doesn't exist, is unauthorized, or is forbidden for url 'https://download.virtualbox.org/virtualbox/6.1.2/VirtualBox-6.1.2-135662-Win.exe'. Exception calling "GetResponse" with "0" argument(s): "The remote server returned an error: (404) Not Found."
The upgrade of virtualbox was NOT successful.
Error while running 'C:\ProgramData\chocolatey\lib\virtualbox\chocolateyInstall.ps1'.
 See log for details.
```

## Possible Solution

The virtualbox URL has changed to https://download.virtualbox.org/virtualbox/6.1.2/VirtualBox-6.1.2-135663-Win.exe

**Comment by AdmiringWorm** [comment]
This had already been reported and responded to before you opened this issue, please see https://chocolatey.org/packages/virtualbox#comment-4759092851

TL;DR: A fix version is already pushed that fixes this issue.
